src/preprocessing/preprocessing_pass_context.h

```diff
@@ -59,6 +60,7 @@ class PreprocessingPassContext
 
   /** Instance of the ITE remover */
   RemoveTermFormulas* d_iteRemover;
+
```

Comment:
Is there any reason for adding a newline here?

Reply:
No, that's a left over, I'll remove.